test/runtests.jl

```diff
@@ -98,12 +98,12 @@ for dev in devices
     @test test_bqgql_advection(0.0005, "ForwardEuler", dev)
     @test test_bqgql_energyenstrophy(dev)
     @test test_bqgql_problemtype(dev, Float32)
-    @test BarotropicQGQL.nothingfunction() == nothing
+    @test isnohting(BarotropicQGQL.nothingfunction())
```

Comment:
typo here

but I'd urge to keep the PR to its title?

Reply:
Yep, will do.